workers/python/Dockerfile

```diff
@@ -67,10 +67,10 @@ COPY --from=builder-base $PYSETUP_PATH $PYSETUP_PATH
 
 WORKDIR /app
 
-COPY workers/python/python/ .
+# Prepare worker resources.
+COPY workers/python/python/init_resources.py .
+RUN python3.11 init_resources.py    
 
-# ENV GLOBAL_WORKING_DIR_ENABLED true
-# ENV GLOBAL_WORKING_DIR /tmp/unstructured
-# ENV HUGGINGFACE_HUB_CACHE /tmp/huggingface
+COPY workers/python/python/ .
 
-CMD [ "python3.11", "main.py"]
+CMD ["python3.11", "main.py"]
```

Comment:
Unrelated: please change default healthcheck port and make it configurable through env, we have mlflow running on 8080 in bee-stack

https://github.com/i-am-bee/bee-api/blob/549f924e6203557dfac4a392a0907f35f9ed5540/workers/python/python/main.py#L63

Reply:
https://github.com/i-am-bee/bee-api/pull/83